src/app/features/sportsandsports/index.js

```
@@ -0,0 +1,270 @@
+      .addCase(getSportsAndSportsCategories.fulfilled, (state, action) => {
+        state.isFetching = false;
+        state.categories = action?.payload;
+      })
+      .addCase(getSportsAndSportsCategories.rejected, (state, action) => {
+        state.isFetching = false;
+        state.error = action?.payload;
+      })
+      .addCase(getSportsAndSportsByCategory.pending, (state) => {
+        state.isSportsFetching = true;
+      })
+      .addCase(getSportsAndSportsByCategory.fulfilled, (state, action) => {
```

ESLint

[error] 'action' is defined but never used.

(no-unused-vars)